lsr_role2collection.py

```diff
@@ -963,13 +965,19 @@ def role2collection():
     module_utils_dir = plugin_dir / "module_utils"
     docs_dir = dest_path / "docs"
 
-    _src_path = args.src_path.resolve()
+    src_path = args.src_path.resolve()
     src_owner = args.src_owner
     if not src_owner:
-        src_owner = os.path.basename(_src_path)
-    src_path = Path(_src_path) / role
-    if not src_path.exists():
-        print(f"Error: {src_path} does not exists.")
+        src_owner = os.path.basename(src_path)
+    _meta_path = src_path / "meta/main.yml"
```

Comment:
Are you using the existence of `meta/main.yml` to determine if the directory contains a role?  If so, then according to Ansible the bare minimum for a role is `tasks/main.yml` - ansible-lint also uses this to determine if the directory is a role or not.

Reply:
Ok. Let me replace it with tasks. I thought it is common to check meta/main.yml looking at this line. :)
https://github.com/linux-system-roles/test-harness/blob/master/test/run-tests#L375

Reply:
that line is wrong too . . . https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/utils.py

Reply:
Shall I open another PR for the line? or is it ok to include the fix in https://github.com/linux-system-roles/test-harness/pull/113?

Reply:
> Shall I open another PR for the line? or is it ok to include the fix in [linux-system-roles/test-harness#113](https://github.com/linux-system-roles/test-harness/pull/113)?

I think this PR

Reply:
Ah, test-harness intentionally uses meta/main.yml to check meta info in checkmeta.py.
So, they are good with meta.